kolibri/plugins/coach/assets/src/app.js

```diff
@@ -53,9 +54,23 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXAMS,
-          path: '/:class_id/exams',
+          path: '/:classId/exams',
           handler: (toRoute, fromRoute) => {
-            actions.showExamsPage(store, toRoute.params.class_id);
+            examActions.showExamsPage(store, toRoute.params.classId);
+          },
+        },
+        {
+          name: PageNames.CREATE_EXAM,
```

Comment:
I made the call to move the subpage logic to the action for 'recent' reports, but I'm honestly still a bit torn. There, the pages seemed more "similar", whereas in this case "new exam" vs "list exams" are clearly different pages. Maybe I should have left the 'recent' pages alone.

The root problem is that all our sub-pages, -routes, -states, etc are being mixed together in one big namespace. My hope is that we can address all this ugliness with a move to vuex2 modules plus sub-routers. That is refactor is tentatively slated for 0.6.x.